src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-bar bar--0" data-qa="big">
```

Comment:
please don't use numbers in class names. fix it everywhere
```suggestion
    <form class="search-bar bar--big" data-qa="big">
```

Reply:
Not fixed
<img width="851" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/9dda2c4e-7cb5-418d-adc6-f395a842fff7">